src/App/App.js

```diff
@@ -170,42 +164,38 @@ class App extends Component {
     this.setState( {cardAreaView: view})
   }
 
-  removeFaveListItem = (indexToRemove) => {
-    const [...newFaveDecks] = this.state.userFaveDecks;
-    newFaveDecks.splice(indexToRemove, 1)
-    this.setState({ userFaveDecks: newFaveDecks },
-      localStorage.setItem('userFaveDecks', JSON.stringify(newFaveDecks)));
+  addToWishlist = (card) => {
+    card.wishListCount = 1;
+    const newWishList = this.state.wishList.concat([card]);
+    this.saveArray('wishList', newWishList);
   }
 
-  removeWishListItem = (indexToRemove) => {
```

Comment:
These two methods were almost identical. I removed them, added the splicing logic in their respective list item components, and had the child components call `saveArray()`.